pkgs/servers/unftp/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, linux-pam
+, rustPlatform
+, fetchFromGitHub
+, buildFeatures ? [ "gnu" ]
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "unFTP";
```

```suggestion
  pname = "unftp";
```

`pname` must not contain uppercase letters.